e2e/TROUBLESHOOTING.md

```diff
@@ -5,10 +5,88 @@ The first step in determining the exact failure is to try and reproduce the test
+
+If you want to recreate the stack, please run the following command, which will run the tests but without any valid scenario (see `TAGS` variable):
+
+```shell
+# login as root
+sudo su -
+# stop previous stack
+docker-compose -f ~/.op/compose/profiles/fleet/docker-compose.yml down --remove-orphans
+# bootstrap the new stack
+TAGS="non-existing-tag" TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE DEVELOPER_MODE=true make -C e2e/_suites/fleet functional-test
+```
+
+#### The Agent VMs
+There different VM flavours that you can use to run the Elastic Agent and enrol it into the Stack: Debian, CentOS, SLES15, Oracle Linux... using AMD and ARM as architecture. You can find the full reference of the platform support [here](https://github.com/elastic/e2e-testing/blob/4517dfa134844f720139d6bab3955cc8d9c6685c/.ci/.e2e-platforms.yaml#L2-L42).
+
+In these VMs, the test framework will download a binary to install the Elastic Agent (TAR files, DEB/RPM packages...), and will execute the different agent commands to install, enrol, uninstall, etc.
+
+#### Getting SSH access to the VMs
+To access the machines, you must be allowed to do so first, and for that, please submit a PR adding your Github username in alphabetical order to [this file](../.ci/ansible/github-ssh-keys), keeping a blank line as file ending. The user to access each EC2 used on the tests can be found [here](https://github.com/elastic/e2e-testing/blob/main/.ci/.e2e-platforms.yaml). When submitting the pul request with your user, please remember to add the right backport labels (ex. `backport-v8.2.0`) so that you will be able to SSH into the supported maintenance branches.
+
+To get the IP address of the VMs, please go to the Jenkins'm BlueOcean UI of the job you manually triggered with `DEVELOPER_MODE=true`, and look up the **Deploy Test Infra** stage. Under its steps, you will see different Ansible executions to provision the VM. Please open any of them and look for any log entry containing an IP address:
+
+```shell
+[2022-04-21T05:20:27.133Z] TASK [Gathering Facts] *********************************************************
+[2022-04-21T05:20:29.541Z] ok: [3.144.74.102]
+```
+
+The IP address of that VM is `3.144.74.102`.
+
+For the agent VMs it's exactly the same, but looking up the **parallel stages** right after the Test Infra. Again, look for any Ansible task containing an IP address, as shown above.
+
+Once you have both IP addresses, one for the stack and one for the agent in the OS/Arch you are interested in, please open two terminals, one for each. Then SSH into the machines using: 1) the public SSH key you have in your Github account, and 2) the right user for the machine, as described [here](https://github.com/elastic/e2e-testing/blob/4517dfa134844f720139d6bab3955cc8d9c6685c/.ci/.e2e-platforms.yaml#L2-L42).
```

Comment:
[Suggestion]
It's in part personal opinion, but I find it easier to read when the items are each one on a separated line:

```suggestion
Once you have both IP addresses, one for the stack and one for the agent in the OS/Arch you are interested in, please open two terminals, one for each. Then SSH into the machines using: 
1) the public SSH key you have in your Github account, and 
2) the right user for the machine, as described [here](https://github.com/elastic/e2e-testing/blob/4517dfa134844f720139d6bab3955cc8d9c6685c/.ci/.e2e-platforms.yaml#L2-L42).
```